homeassistant/components/fritzbox/light.py

```diff
@@ -72,8 +72,10 @@ def __init__(
         """Initialize the FritzboxLight entity."""
         super().__init__(coordinator, ain, None)
 
-        self._attr_max_color_temp_kelvin = int(max(supported_color_temps))
-        self._attr_min_color_temp_kelvin = int(min(supported_color_temps))
+        if supported_color_temps:
```

Comment:
Can we break out the light changes to a separate PR and just bump the library version in this PR?

Reply:
Hey, if you want to i can create a seperate PR for the bump, however the changes in the version lead to a faulty integration, where none of the bulbs are detected correctly anymore.

Reply:
Then we should do it in the same PR. Only if we can bump without other changes and still have a working integration should we make two PRs.